core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -646,7 +646,7 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
    * Executes this effect, skipping the error but returning optionally the success.
    */
   final def option: ZIO[R, Nothing, Option[A]] =
-    self.foldCauseM(_ => IO.succeed(None), a => IO.succeed(Some(a)))
+    self.foldM(_ => IO.succeed(None), a => IO.succeed(Some(a)))
```

Comment:
This looks good, can you make sure `either` is consistent with this?

Reply:
Just checked and `either` is implemented in terms of `foldM` versus `foldCauseM`, so I think we are good here. Also reinforces that this is the right way to go as these methods should be consistent.